jujugui/static/gui/src/app/components/deployment/deployment.js

```diff
@@ -171,6 +170,7 @@ YUI.add('deployment-component', function() {
   requires: [
     'deployment-add-credentials',
     'deployment-choose-cloud',
+    'deployment-finalise-account',
```

Comment:
I'm not sure what this was supposed to reference but it's throwing a YUI module warning because it doesn't exist.